pvnet_app/config.py

```diff
@@ -54,38 +54,146 @@ def overwrite_config_dropouts(config: dict) -> dict:
 
     Args:
         config: The data config
+
     """
 
     # Replace data sources
     for source in ["satellite"]:
         if source in config["input_data"]:
-            if config["input_data"][source][f"{source}_zarr_path"] != "":
-                config["input_data"][source][f"dropout_timedeltas_minutes"] = None
+
+            satellite_config = config["input_data"][source]
+
+            if satellite_config[f"{source}_zarr_path"] != "":
+                satellite_config[f"dropout_timedeltas_minutes"] = None
+                satellite_config[f"dropout_fraction"] = 0
 
     # NWP is nested so much be treated separately
     if "nwp" in config["input_data"]:
         nwp_config = config["input_data"]["nwp"]
         for nwp_source in nwp_config.keys():
             if nwp_config[nwp_source]["nwp_zarr_path"] != "":
                 nwp_config[nwp_source]["dropout_timedeltas_minutes"] = None
+                nwp_config[nwp_source]["dropout_fraction"] = 0
+
+    return config
+
+
+def reformat_config_data_sampler(config: dict) -> dict:
+    """Reformat config
+
+    This is to keep the configurations from ocf-data-sampler==0.0.19 working,
+    we need to upgrade them a bit to the configuration in ocf-data-sampler>=0.1.5
+
+    Args:
+        config: The data config
+
+    """
+
+    # Replace data sources
+    source = "satellite"
+    if source in config["input_data"]:
```

Comment:
Could just be
```suggestion
    if "satellite" in config["input_data"]:
```

 but if by naming it source it makes it clearer then maybe we should have it consistent for all inputs as the NWP one doesn't do this